apps/gnome-mobile/src/app/(app)/collection.tsx

```diff
@@ -1,13 +1,42 @@
-import { Tabs } from "expo-router";
-import React from "react";
-import { Text, View } from "react-native";
-
-const Collection = () => {
-  return (
-    <View>
-      <Text>Kolekcja</Text>
-    </View>
-  );
-};
-
-export default Collection;
+      import React, { useState } from "react";
+      import { Text, View, FlatList, StyleSheet } from "react-native";
+      import axios from "axios";
+      import { Button } from "@/components/ui/button";
+      
+      const API_URL = "http://localhost:3000/api/rest/v1/gnomes"; // Zmiana na emulator-friendly URL
+      
+      const Collection = () => {
+        const [krasnale, setKrasnale] = useState([]);
+      
+        const fetchKrasnale = async () => {
+          try {
+            const response = await axios.get(API_URL);
+            setKrasnale(response.data);
+          } catch (error) {
+            console.error("Błąd pobierania danych:", error.response ? error.response.data : error.message);
+          }
+        };
```

Comment:
tu bym się przyczepił tego jak wykorzystany jest axios, ogólnie to poprzednio korzystaliśmy z instancji axiosa która ten localhost:3000 miała już ustawiony

starczy zostawić: axiosInstance.get("/api/rest/v1/gnomes")